src/WebAPI/ClientApp/src/lang/de_DE.json

```diff
@@ -0,0 +1,416 @@
+				"text": "Der Benutzername des Plex Kontos"
+			},
+			"password": {
+				"label": "Passwort",
+				"title": "Passwort",
+				"text": "Das Passwort des Plex Kontos."
+			}
+		},
+
+		"settings": {
+			"paths": {
+				"download-folder": {
+					"label": "Download Pfad",
+					"title": "Download Pfad",
+					"text": "In diesen Pfad werden alle teile eines Downloads heruntergeladen und zusammengeführt bevor sie in das Zielverzeichnis verschoben werden."
+				},
+				"movie-folder": {
+					"label": "Filme Pfad",
+					"title": "Filme Pfad",
+					"text": "Standartpfad für Filme wenn kein individueller Pfad angegeben wurde."
+				},
+				"tv-show-folder": {
+					"label": "Serien Pfad",
+					"title": "Serien Pfad",
+					"text": "Standartpfad für Serien wenn kein individueller Pfad angegeben wurde."
+				}
+			},
+			"ui": {
+				"language": {
+					"language-selection": {
+						"label": "Sprache",
+						"title": "",
+						"text": ""
+					}
+				},
+				"date-and-time": {
+					"short-date-format": {
+						"label": "Datum kurzformat",
+						"title": "",
+						"text": ""
+					},
+					"long-date-format": {
+						"label": "Dazum langformat",
+						"title": "",
+						"text": ""
+					},
+					"time-zone": {
+						"label": "Zeitzone",
+						"title": "",
+						"text": ""
+					},
+					"time-format": {
+						"label": "Zeit Format",
+						"title": "",
+						"text": ""
+					},
+					"show-relative-dates": {
+						"label": "Zeige Relative Daten",
+						"title": "",
+						"text": ""
+					}
+				},
+				"confirmation-settings": {
+					"download-movie-confirmation": {
+						"label": "Frage nach Bestätigung für Filme Download",
+						"title": "Frage nach Bestätigung für Filme Download",
+						"text": ""
+					},
+					"download-tvshow-confirmation": {
+						"label": "Frage nach Bestätigung für Serien Download",
+						"title": "Frage nach Bestätigung für Serien Download",
+						"text": ""
+					},
+					"download-season-confirmation": {
+						"label": "Frage nach Bestätigung für Staffel Download",
+						"title": "Frage nach Bestätigung für Staffel Download",
+						"text": ""
+					},
+					"download-episode-confirmation": {
+						"label": "Frage nach Bestätigung für Folgen Download",
+						"title": "Frage nach Bestätigung für Folgen Download",
+						"text": ""
+					}
+				}
+			},
+			"advanced": {
+				"download-manager-section": {
+					"download-segments": {
+						"label": "Download Teile",
+						"title": "Download Teile",
+						"text": "PlexRipper Multithread-Downloads und kann Medien, die heruntergeladen werden sollen, in individuellen teilen herunterladen und wieder zusammen führen. Das führt in der regel zu einer höheren Downloadgesschwindigkeit aber benötigt auch etwas mehr Zeit um die Teile wieder zusammenzusetzen.\n\nDas setzen auf den Wert 1 deaktiviert die Funktion und lädt die Datei am Stück herunter. Die Anzahl der Segmente hat keinen Einfluss auf die Dauer des zusammen setzens."
```

Comment:
"PlexRipper unterstützt Multithread-Downloads und kann Medien, die heruntergeladen werden sollen, in Segmente aufteilen und individuell herunterladen. Das führt in der Regel zu einer höheren Downloadgeschwindigkeit, benötigt jedoch zusätzlich Zeit um die Segmente  wieder zusammenzusetzen.\n\nDas Setzen auf den Wert 1 deaktiviert die Funktion und lädt die Datei am Stück herunter. Die Anzahl der Segmente hat keinen Einfluss auf die Dauer des Zusammensetzens."